Makefile

```diff
@@ -21,6 +22,24 @@ mypy:
 docs-%:
 	make -C docs $*
 
+refresh_all_requirements:
```

Comment:
the individual make commands work fine, this one throws an error for me. 

```bash
$ make refresh_all_requirements 
[ ! -e .requirements-env ] || exit 1
virtualenv .REQUIREMENTS.txt.in-env
/bin/bash: virtualenv: command not found
make[1]: *** [REQUIREMENTS.txt] Error 127
make: *** [refresh_all_requirements] Error 2
(.venv) 
``` 

I used a pre-existing virtualenv, although it was set up according to your commands below. Could that contribute? 

I'm inside the virtualenv when I call this, as well.

Reply:
Let me try an alternative.